presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -215,16 +215,137 @@ The connector maps PrestoDB types to the corresponding MySQL types:
+
+CREATE SCHEMA
+^^^^^^^^^^^^^
+
+Create a new MySQL schema named ``web``:
+
+.. code-block:: sql
+
+    CREATE SCHEMA mysql.web;
+
+However, please note the following:
+
+.. error::
+
+    This connector does not support creating schemas
+
+CREATE TABLE
+^^^^^^^^^^^^
+
+Create a new MySQL table named ``page_views`` in the ``web`` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views (
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    );
+
+CREATE VIEW
+^^^^^^^^^^^
+
+Create a view named ``view_page_views`` for the ``mysql.web.page_views`` table:
+
+.. code-block:: sql
+
+    CREATE VIEW mysql.web.view_page_views AS SELECT user_id, country FROM mysql.web.page_views;
+
+However, please note the following:
+
+.. error::
+
+    This connector does not support creating views
+
+
+INSERT INTO
+^^^^^^^^^^^
+
+Insert data into the ``page_views`` table:
+
+.. code-block:: sql
+
+    INSERT INTO mysql.web.page_views VALUES(1, 'https://example.com', current_date, 'country');
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views_new AS SELECT * FROM mysql.web.page_views;
+
+SELECT
+^^^^^^
+
+SELECT table operations are supported:
+
+.. code-block:: sql
+
+    SELECT * FROM mysql.web.page_views;
+
+ALTER TABLE
+^^^^^^^^^^^
+
+Alter table operations are supported:
+
+.. code-block:: sql
+
+    ALTER TABLE mysql.web.page_views ADD COLUMN zipcode VARCHAR;
+    ALTER TABLE mysql.web.page_views RENAME COLUMN zipcode TO location;
+    ALTER TABLE mysql.web.page_views DROP COLUMN location;
+
+TRUNCATE
+^^^^^^^^
+
+The MySQL connector can delete all of the data from tables without dropping the table:
+
+.. code-block:: sql
+
+    TRUNCATE TABLE mysql.web.page_views;
+
+DROP TABLE
+^^^^^^^^^^
+
+Drop the table ``page_views``:
+
+.. code-block:: sql
+
+    DROP TABLE mysql.web.page_views;
+
+However, please note the following:
+
+.. error::
+
+    DROP TABLE is disabled in this catalog
+
 MySQL Connector Limitations
 ---------------------------
 
-The following SQL statements are not yet supported:
+The following SQL statements are not supported:
 
 * :doc:`/sql/delete`
 * :doc:`/sql/alter-table`
-* :doc:`/sql/create-table` (:doc:`/sql/create-table-as` is supported)
+* :doc:`/sql/create-table` with PRIMARY KEY
+* :doc:`/sql/create-view`
 * :doc:`/sql/grant`
 * :doc:`/sql/revoke`
 * :doc:`/sql/show-grants`
 * :doc:`/sql/show-roles`
 * :doc:`/sql/show-role-grants`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/update`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/analyze`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-view`
+
+The MySQL connector does not support advanced SQL features such as window functions and data compression in the ``CREATE TABLE AS SELECT`` statement.
```

Comment:
```suggestion
Advanced SQL features such as window functions and data compression are not supported in the ``CREATE TABLE AS SELECT`` statement.
```

This doesn't seem as though it's in the correct place: the CREATE TABLE AS SELECT statement is supported as you show in the example beginning in line 277, this is a limitation of support for that statement - not that the CREATE TABLE AS SELECT is unsupported in its entirety. 

For consistency with the ERROR blocks you have in the examples for CREATE SCHEMA, CREATE VIEW, and DROP VIEW, it seems that this statement belongs in a Note block at the end of the CREATE TABLE AS SELECT example that begins on line 277. 

If you move this sentence into a Note block at the end of the CREATE TABLE AS SELECT example it can be shortened further, as follows: 

"Advanced SQL features such as window functions and data compression are not supported for this SQL operation."